python/pyarrow/compute.py

```diff
@@ -338,6 +339,8 @@ def _make_global_functions():
 
 
 _make_global_functions()
+# Alias for consistency
+utf8_zfill = utf8_zero_fill = globals()["utf8_zero_fill"]
```

Comment:
This can be simplified to:
```suggestion
utf8_zfill = utf8_zero_fill
```

Reply:
Ah, I see. Can you add a comment that it's required for linting, then?